packages/node/src/protocol/utils/get-outcome-increments.ts

```diff
@@ -4,14 +4,16 @@ import {
   OutcomeType,
   SingleAssetTwoPartyCoinTransferInterpreterParams,
   TwoPartyFixedOutcome,
-  TwoPartyFixedOutcomeInterpreterParams
+  TwoPartyFixedOutcomeInterpreterParams,
+  multiAssetMultiPartyCoinTransferEncoding
 } from "@counterfactual/types";
 import { BaseProvider } from "ethers/providers";
 import { BigNumber, defaultAbiCoder } from "ethers/utils";
 
 import { AppInstance } from "../../models";
 import {
   CoinTransfer,
+  convertCoinTransfersToCoinTransfersMap,
```

Comment:
Since this turns out to be a more generic function than just used for the `FreeBalanceApp`, perhaps you should extract it into a more general purpose utils file?